### PR TITLE
World Decomp & Dedupe

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 90 functions (+1.42%) on `us`.

✅ This PR fixes 4 warnings!

File: src/world/area_hos/hos_00/A0BCF0.c [hunk 1]
```diff
@@ -1,3 +1,10 @@
 #include "hos_00.h"
 
-INCLUDE_ASM(s32, "world/area_hos/hos_00/A0BCF0", func_80240000_A0BCF0);
+extern char D_800D9668[];
+
+ApiStatus func_80240000_A0BCF0(Evt* script, s32 isInitialCall) {
+    if (gGameStatusPtr->entryID == 3) {
+        sprintf(&D_800D9668, "hos_bg\0\0\0\0\0\0\0\0\0");
```
Comment (nanaian) [inline src/world/area_hos/hos_00/A0BCF0.c:7]: I hope you can remove these nops?

File: src/world/common/UnkNpcAIMainFunc5.inc.c [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        case 99:
+            func_8004A73C(script);
+            break;
+    }
+
+    if (enemy->varTable[7] == 6) {
+        if (enemy->varTable[8] != 0) {
+            enemy->unk_B5 = 1;
+        } else {
+            enemy->unk_B5 = 0;
```
Comment (nanaian) [inline src/world/common/UnkNpcAIMainFunc5.inc.c:117]: Looks very boolish to me

File: src/world/area_hos/hos_00/A0BCF0.c (hunk 1 above)
Comment (ethteck) [reply src/world/area_hos/hos_00/A0BCF0.c:7]: No, I specifically had to add them to get appropriate padding to happen :/

Comment (nanaian) [reply src/world/area_hos/hos_00/A0BCF0.c:7]: 😭

**Review by nanaian — APPROVED** [review]

**Pull request merged**